src/charm.py

```diff
@@ -170,8 +178,8 @@ def _generate_ue_config(self) -> str:
             key=self._charm_config.key,
             opc=self._charm_config.opc,
             dnn=self._charm_config.dnn,
-            sst=self._charm_config.sst,
-            sd=self._charm_config.sd,
+            sst=self.rfsim_requirer.sst,  # type: ignore[arg-type]
```

Comment:
is there a way to fix this static issue instead of ignoring it ?

Reply:
Static issue is fixed.